### DISCUSSION TITLE
Errors while try to export project

### DISCUSSION THREAD

**Discussion by harvi2048** [timrogers/gh-migrate-project#67] [Q&A] [answered]
Hi,

i`m trying to export project from repository, but getting below errors:
Request failed due to following response errors:
 - Field 'description' doesn't exist on type 'ProjectV2SingleSelectFieldOption'
 - Field 'color' doesn't exist on type 'ProjectV2SingleSelectFieldOption'
Can you guide me what can be the reason for this.
The export is done from GHES.

Regards,
Stefan

**Comment by harvi2048** [comment]
Sharing some screenshot
![image](https://github.com/timrogers/gh-migrate-project/assets/71321238/86b5d152-db7d-4750-bcda-8868ad0f5af5)

**Comment by timrogers** [comment]
Hi Stefan! Can you confirm which Enterprise Server version you’re using? I didn’t test against older versions, and I’m guessing there may be GraphQL differences between versions!

Sent from Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by harvi2048** [comment]
Hi Tim, our Enterprise Server version is Version 3.9.5 665f58f.

**Reply by timrogers** [reply]
Thanks! Handling different Server versions will add some complexity, but I'm confident that I can find a graceful way to do it. I'll let you know once we have a fix.

**Reply by harvi2048** [reply]
Will be very grateful if you find solution to the issue :)

**Comment by timrogers** [comment, marked as answer]
@harvi2048 Can you try [v0.3.0](https://github.com/timrogers/gh-migrate-project/releases/tag/v0.3.0)? Exports should be fixed in this version based on my testing. 

At the moment, only imports to GitHub.com are supported, but I will look at how we can support imports to GHES. Sadly, the APIs are not compatible.

**Comment by harvi2048** [comment]
@timrogers thanks for the info, will try and let you know. We will import the Projects to github.com.

Regards,
Stefan

**Comment by harvi2048** [comment]
Hi @timrogers ,

the export is running fine with v0.3.0, but now the import seems to have issue.
It have stick for all night after pressing Enter on this message:
Press Enter to continue...

Attaching verbose log from the import. The Project seems to be created on the target destination, but is empty.
[import_log.txt](https://github.com/timrogers/gh-migrate-project/files/13389640/import_log.txt)
Before pressing Enter, the missing "Status" fields was added to the newly created project.

Regards,
Stefan

**Reply by timrogers** [reply]
That's weird! I'm surprised that broke as I don't think I changed any code relevant to this part of the import flow. 

Based on your logs, it seems to be totally stuck because there are no GraphQL requests being logged. 

If you haven't already, could you try again, just to rule out a transient issue?

If that doesn't work, I can take a look. 

Sorry for the pain here - this tool has worked for me and some others, but your pain proves that I need to add some automated integration tests to make sure it actually works. I'll do that next week.

**Reply by harvi2048** [reply]
The log is from my second attempt, the first one have stuck on the same position.

**Reply by timrogers** [reply]
Interesting! I'm not able to reproduce this. What operating system are you running? My thinking is that something is going wrong with the prompt where you have to press Enter.

**Comment by harvi2048** [comment]
I`m using OS Name	Microsoft Windows 10 Enterprise and starting the process from Git bash terminal. Let me try from Linux or Windows CMD

**Reply by timrogers** [reply]
That's really interesting. Let me know how you get on. I suspect some kind of Git Bash incompatibility!

**Reply by harvi2048** [reply]
After using Windows CMD, the project was imported!
Thanks for the support and professionality Tim.

**Reply by timrogers** [reply]
Thanks! I'm going to close this discussion, and I will create a follow-up issue regarding Git Bash.
